Lib/fontTools/varLib/interpolatable.py

```diff
@@ -159,17 +159,25 @@ def test(glyphsets, glyphs=None, names=None):
 
 def main(args):
```

Comment:
let's make sure all main() signatures look like this:

```suggestion
def main(args=None):
```

to have argparse parse_args automatically get sys.argv when None, and being able to call main([...]) with a list of string arguments.

in `if __name__ == "__main__"` below you then only need to call `main()` with no args